src/hotspot/cpu/aarch64/templateInterpreterGenerator_aarch64.cpp

```diff
@@ -994,7 +994,7 @@ address TemplateInterpreterGenerator::generate_CRC32_updateBytes_entry(AbstractI
     // Arguments are reversed on java expression stack
     // Calculate address of start element
     if (kind == Interpreter::java_util_zip_CRC32_updateByteBuffer) {
-      __ ldr(buf, Address(esp, 2*wordSize)); // long buf
+      __ ldr(buf, Address(esp, 2*wordSize)); // jlong buf
```

Comment:
Maybe just drop the type in the comment?  Similarly in a couple later places.

Reply:
Ok. That'll give me less 'long' matches.

Reply:
No, it's useful because it describes the parameter that you're loading.  I'm leaving this jlong.

Reply:
Or I'll revert this whole file.